### PR TITLE
Add dispatch label to automatic PR labeler

### PR DESCRIPTION
**Pull request by jrbourbeau**
Following up on https://github.com/dask/dask/pull/7503 and https://github.com/dask/dask/pull/7505, this PR adds a `dispatch` label to our automatic labeler CI job

### CODE REVIEW

File: dask/dataframe/dispatch.py [hunk 1]
```diff
@@ -47,6 +47,9 @@ def concat(
         Whether to ignore the order when doing the union of categoricals.
         Default is False.
     """
+
+    print("~~ James was here ~~")
```
Comment (quasiben) [inline dask/dataframe/dispatch.py:51]: Should this be removed ?

**Comment by jsignell** [comment]
I think the label has to exist.

File: dask/dataframe/dispatch.py (hunk 1 above)
Comment (jrbourbeau) [reply dask/dataframe/dispatch.py:51]: Yeah, definitely. I've just temporarily added this to try and trigger the automatic `dispatch` label

**Comment by jrbourbeau** [comment]
Hrm I'm having some trouble getting the labeler to work for specific files. @jsignell do you've worked with the auto-labeler before. Any idea what I might be doing wrong here?

**Review by jsignell — COMMENTED** [review]
I should have thought of this earlier but the labeler.yml probably needs to be on main in order to work.

**Comment by jsignell** [comment]
I think I made a PR on my fork when I was first testing out labeling

**Comment by jsignell** [comment]
Yeah it looks like that's the deal https://github.com/dask/dask/pull/7506

**Comment by jrbourbeau** [comment]
Ah, I see -- thanks for the tip @jsignell! I tested this out on my fork and it seems to be working 

Screen Shot 2021-06-04 at 5 54 27 PM

**Pull request merged**